src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0,05);
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.header__menu {
+  list-style: none;
+  height: 100%;
+}
+
+.header__li {
+  display: inline-block;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  margin-left: 16px;
+  position: relative;
+}
+
+.header__li:first-child {
+  margin-left: 0;
+}
+
+.header__link {
+  color: #000;
+  text-decoration: none;
+  transition: 0.5s;
+  padding: 22.5px 0;
```

Don't use fractions of pixels, it's a bad practice
```suggestion
  padding: 23px 0;
```
or

```suggestion
  padding: 22px 0;
```